packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -166,6 +166,13 @@ function createDefaultRegistry(): Registry {
   return new Registry([...defaultStargateTypes, ...wasmTypes]);
 }
 
+function createDefaultAminoTypes(): AminoTypes {
```

Comment:
Do we need those function? IMO it just created indirection and we don't want to reuse them in multiple places

Reply:
Nope, I agree. I would love to clean those up, didn't want to make any architecture decisions on my own :)